chia/cmds/data.py

```diff
@@ -106,6 +106,26 @@ def create_fee_option() -> Callable[[FC], FC]:
     )
 
 
+def create_page_option() -> Callable[[FC], FC]:
+    return click.option(
+        "-p",
+        "--page",
+        help="Enables pagination of the output and requests a specific page.",
+        type=int,
+        required=False,
+    )
+
+
+def create_max_page_size_option() -> Callable[[FC], FC]:
+    return click.option(
+        "-mp",
+        "--max_page_size",
```

Comment:
@emlowe, where are we at with multi-character short options and underscores?

Reply:
multi-character short options should be avoided, i'd prefer not having one if we can't think of a single letter option.
let's use use hypens - we use hypens mostly, although there isn't much consistency. so `--max-page-size`

Reply:
Looks like we still have `-mp`.